HAST-295: Adding license and elaborating Readme (#6)
Lombiq/Hastlayer-Remote-Bridge: Readme.md

# Hastlayer - Remote Bridge

Project that contains everything necessary for all the components of the remote Hastlayer infrastructure.

> DEPRECATED! This repository was ued before we open-sourced the whole project. That removed the necessity for remote transformation.

Reviewer: Typo. Also if I understand correctly we are removing the remote services entirely. So it's not just that this is unnecessary, but no longer possible even if someone wanted to do it, right?

```suggestion
DEPRECATED! This repository was used before we open-sourced the whole project. We no longer support remote transformation.
```

Author: Yep!
